### PR TITLE
[IE PYTHON] update license 

### PR DESCRIPTION
**Pull request by AlexeyLebedev1**
### Tickets:
   - 49448

### CODE REVIEW

File: inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx [hunk 1]
```diff
@@ -1,3 +1,21 @@
+
```
Comment (akuporos) [inline inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx:1]: Please, remove blank lines before the license

File: inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pxd [hunk 2]
```diff
@@ -1,3 +1,21 @@
+
```
Comment (akuporos) [inline inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pxd:1]: also, remove this blank line, please

**Review by akuporos — CHANGES_REQUESTED** [review]
Please, remove the first blank lines and undo changes in inference-engine/thirdparty/mkl-dnn

**Comment by ilyachur** [comment]
Could you fill the PR description?

**Review by akuporos — APPROVED** [review]
LGTM

**Review by nkogteva — APPROVED** [review]

**Pull request merged**